pkg/controller/metrics/adapters/metrics_sender_to_prometheus.go

```diff
@@ -0,0 +1,106 @@
+package adapters
+
+import (
+	log "github.com/golang/glog"
+	"github.com/opensds/opensds/pkg/model"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/push"
+	"os"
+	"strconv"
+	"time"
+)
+
+var nodeExporterFolder = "/root/prom_nodeexporter_folder/"
```

Comment:
Does this have to be fixed?

Reply:
no...will be configurable, will do as part of https://github.com/opensds/opensds/issues/691